README.md

```diff
@@ -1,72 +1,51 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# To Do list
 
-> Description the project.
+To do project is webpack project that list activities someone can do at a specific time 
 
+In this TO-DO list, you can add or remove you activities you can block completed activities if theyy are any and you can remove all completed activities at once
 
-## Built With
+## Built With:
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML
+- CSS
+- JAVASCRIPT
+- webpack
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
```

Comment:
Kindly use the correct link for the live demo so that people can test your site.